test/functional/lua/vim_spec.lua

```diff
@@ -1071,15 +1071,15 @@ describe('lua stdlib', function()
     ]])
     )
 
-    -- Fix github issue #23654
-    ok(exec_lua([[
-      local a = { sub = { [1] = 'a' } }
-      local b = { sub = { b = 'a' } }
-      local c = vim.tbl_deep_extend('force', a, b)
-      return vim.deep_equal(c, { sub = { [1] = 'a', b = 'a' } })
-    ]]))
-
-    matches('invalid "behavior": nil', pcall_err(exec_lua, [[return vim.tbl_deep_extend()]]))
+    matches(
```

Comment:
this matches() case looks like a style-only change

Reply:
Yes, the revert dropped a style-only change in the meantime. Will re-revert.